calypso/service.go

```diff
@@ -711,36 +757,50 @@ func (s *Service) NewProtocol(tn *onet.TreeNodeInstance, conf *onet.GenericConfi
 
 		// Wait for DKG in reshare mode to end
 		go func(id byzcoin.InstanceID) {
+			// TODO: properly propagate errors during execution of DKG protocol
+			// (see dedis/cothority#2320)
 			<-setupDKG.Finished
+
+			setValidPeers := func(r *onet.Roster) {
+				s.SetValidPeers(s.NewPeerSetID(id[:]), r.List)
```

Comment:
As the argument is always the same, couldn't you just write:
```
setValidPeers := func(){
    s.SetValidPeers(s.NewPeerSetID(id[:]), s.storage.Rosters[id].List)
}
```

Reply:
I could, but I find it clearer to be explicit at the call point.